packaging/helm-charts/helm3/strimzi-kafka-operator/values.yaml

```diff
@@ -72,6 +75,7 @@ podDisruptionBudget:
   # If both values are set, the template will use the first one and ignore the second one. currently by default minAvailable is set to 1
   minAvailable: 1
   maxUnavailable:
+  unhealthyPodEvictionPolicy: AlwaysAllow
```

Comment:
Shouldn't be this `IfHealthyBudget`? 

> The default behavior when no policy is specified corresponds to the IfHealthyBudget policy.

Reply:
That's exactly my same question :-)

Reply:
My understanding of the doc is that `AlwaysAllow` means pods that are `Running` but not yet passing their `readinessProbe` can be evicted. If the pod isn't passing that probe, blocking feels odd to me.

Reply:
You understanding is correct but it's not the default which the document says:

> The default behavior when no policy is specified corresponds to the IfHealthyBudget policy.

Reply:
I did not noticed it ... but +1 for sticking with the Kubernetes default as the Helm Chart default.